### PR TITLE
include text of NEWS.md from relevant PACTA packages in manifest para…

### PR DESCRIPTION
**Pull request by cjyetman**
…meters

### CODE REVIEW

File: run_pacta_data_preparation.R [hunk 1]
```diff
@@ -782,6 +782,19 @@ ent_entity_affiliates_last_update <-
   pull(ent_entity_affiliates_last_update) %>%
   unique()
 
+# include PACTA packages NEWS.md test in the parameters to export
+pacta_packages <- c("pacta.data.preparation", "pacta.scenario.preparation")
```
Comment (jdhoffa) [inline run_pacta_data_preparation.R:786]: WET: These packages are defined again here:
https://github.com/RMI-PACTA/workflow.data.preparation/blob/331f6abf1bbe1ae990960c87a09b8de6a2e63deb/run_pacta_data_preparation.R#L855
and here:
https://github.com/RMI-PACTA/workflow.data.preparation/blob/331f6abf1bbe1ae990960c87a09b8de6a2e63deb/run_pacta_data_preparation.R#L861

for essentially the same purpose. 

Would pose less maintenance burden to define the pacta_packages you want to include NEWS information from once, and use the same argument for both copying over the files, and writing the manifest.

**Review by jdhoffa — CHANGES_REQUESTED** [review]

File: run_pacta_data_preparation.R (hunk 1 above)
Comment (cjyetman) [reply run_pacta_data_preparation.R:786]: agreed... I didn't wanna change anything contextually separate here... let's leverage the new `pacta_packages` once this PR is merged in a new PR that cleans up the `NEWS.md` copying bit

Comment (cjyetman) [reply run_pacta_data_preparation.R:786]: #35

**Review by jdhoffa — APPROVED** [review]
Given #35, this LGTM

**Pull request merged**